docs/DeveloperGuide.md

```diff
@@ -292,10 +300,33 @@ Note: For further details on the `FileHandler` class / `setUpFileHandler()` meth
 
 
 ## Components
+FindOurSEP is primarily a Command-Line Interface (CLI) based Java Application. The entry point to our application is
```

Comment:
Should all of this just move to the top of the DG instead?

Reply:
That works as well